docs/en_us/internal/testing.rst

```diff
@@ -94,6 +94,22 @@ Test Locations
    -  Bok Choy Accessibility Tests: located under ``common/test/acceptance/accessibility``
    -  Bok Choy PageObjects: located under ``common/test/acceptance/pages``
 
+-  Content type examples:
+
+   The Accessibility Mega Course
+   `https://github.com/cptvitamin/demo-test-course
+   <https://github.com/cptvitamin/demo-test-course>`_ provides
+   a comprehensive collection of all edX content types. The course exercises
+   all possible Studio content creation and is a valuable testing resource.
+   It is available by default in sandboxes.
```

Comment:
This is not technically true yet, lets omit it for now and add it back once we get the course included in sandboxes.